ogpf.f90

```diff
@@ -1733,37 +1733,37 @@ subroutine processcmd(this)
         ! write the plot style for data
         ! this is used only when 3D plots (splot, cplot) is used
         if (allocated(this%txtdatastyle)) then
-            write ( this%file_unit, '(a)' ) 'set style data '//this%txtdatastyle  !set data style
-            write ( this%file_unit, '(a)' )  ! emptyline
+            write ( this%file_unit, '("set style data ", a)' ) this%txtdatastyle
+            write ( this%file_unit, '(a)' )
         end if
 
 
         ! Write options
         if ( this%hasoptions ) then
-            write( unit = this%file_unit, fmt= '(a)') ' '
-            write( unit = this%file_unit, fmt= '(a)') '# options'
-            write( unit = this%file_unit, fmt= '(a)') this%txtoptions
-            write ( this%file_unit, '(a)' )  ! emptyline
+            write ( this%file_unit, '(" ")' )
+            write ( this%file_unit, '("# options")' )
+            write ( this%file_unit, '(a)' ) this%txtoptions
+            write ( this%file_unit, '(a)' )

```

Comment:
What is the reason for modifying this section?

Reply:
Consistency: why some write statements have the `unit=` and `fmt=` while others don't?